docs/setup_docs.sh

```diff
@@ -63,7 +63,7 @@ mkdir tmp
 cd tmp
 
 integrate_connector_docs elasticsearch v3.0.0
-integrate_connector_docs aws v3.0.0
+integrate_connector_docs aws v4.0
```

Comment:
v4.0.0? Is the `.0` intentionally left out?

Reply:
`v4.0` is the branch, `v4.0.0` is the tag. I am using the branch because the tag does not render correctly